### PR TITLE
Add code tabs for scala3-book/taste-repl

### PR DESCRIPTION
**Pull request by byyue**
See #2481:
Scala 3 Book -> A Taste of Scala -> The REPL.

### CODE REVIEW

File: _overviews/scala3-book/taste-repl.md [hunk 1]
```diff
@@ -11,46 +11,74 @@ next-page: taste-vars-data-types
...
+
+{% tabs command-line class=tabs-scala-version %}
+
+{% tab 'Scala 3' for=command-line %}
 ```bash
 $ scala
-Welcome to Scala 3.0.0 (OpenJDK 64-Bit Server VM, Java 11.0.9).
-Type in expressions for evaluation.
-Or try :help.
+Welcome to Scala 3.1.3 (1.8.0_322, Java OpenJDK 64-Bit Server VM).
```
Comment (bishabosha) [inline _overviews/scala3-book/taste-repl.md:20]:
```suggestion
Welcome to Scala {{site.scala-3-version}} (1.8.0_322, Java OpenJDK 64-Bit Server VM).
```

File: _overviews/scala3-book/taste-repl.md [hunk 2]
```diff
@@ -11,46 +11,74 @@ next-page: taste-vars-data-types
...
+Type in expressions for evaluation. Or try :help.
 
 scala> _
 ```
+{% endtab %}
+
+{% tab 'Scala 2' for=command-line %}
+```bash
+$ scala
+Welcome to Scala 2.13.8 (OpenJDK 64-Bit Server VM, Java 1.8.0_342).
```
Comment (bishabosha) [inline _overviews/scala3-book/taste-repl.md:30]:
```suggestion
Welcome to Scala {{site.scala-version}} (OpenJDK 64-Bit Server VM, Java 1.8.0_342).
```

**Review by bishabosha — APPROVED** [review]
Thank you very much! The suggestions I made are just to make it more maintainable

**Comment by byyue** [comment]
Thank you for the review! I have another thought: to set the Scala 3 tab as the default, at least in the _Scala 3 Book_. That's why I put it before the Scala 2 tab. 

image

@julienrf Do you consider it wise to unify this kind of format?

File: _overviews/scala3-book/taste-repl.md [hunk 3]
```diff
@@ -11,46 +11,74 @@ next-page: taste-vars-data-types
 The Scala REPL (“Read-Evaluate-Print-Loop”) is a command-line interpreter that you use as a “playground” area to test your Scala code.
 You start a REPL session by running the `scala` or `scala3` command depending on your installation at your operating system command line, where you’ll see a “welcome” prompt like this:
 
+
+{% tabs command-line class=tabs-scala-version %}
+
+{% tab 'Scala 3' for=command-line %}
```
Comment (bishabosha) [inline _overviews/scala3-book/taste-repl.md:17]: I hadn't noticed that the scala 3 tab came first until I saw your comment, for consistency please would you swap the tabs? :) - We will also likely rebrand the book to just the Scala Book when we have updated enough code examples

**Review by bishabosha — CHANGES_REQUESTED** [review]
For me personally numbers ascending left to right across the tabs makes sense, but maybe others would object - but if we chose to change the convention then that should be done globally across the site in a separate PR.

As time goes on we will likely rebrand the book to just the Scala Book when we have updated enough code examples

**Review by bishabosha — APPROVED** [review]

**Comment by bishabosha** [comment]
Thanks a lot!

**Pull request merged**